builtin/game/chatcommands.lua

```diff
@@ -860,7 +865,7 @@ core.register_chatcommand("ban", {
 
 core.register_chatcommand("unban", {
 	params = "<name> | <IP_address>",
-	description = "Remove IP ban",
+	description = "Remove ban from a player",
```

Comment:
`on a player`

Reply:
Or maybe 'Remove player ban'?

Reply:
'Remove player ban'?

Reply:
I honestly feel the original was fine. If you need to use this command, you should know you're removing it from a player. What else do you ban?